pkg/api/helpers.go

```diff
@@ -179,3 +180,65 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	}
 	return c
 }
+
+// GetAccessModesAsString returns a string representation of an array of access modes.
+// modes, when present, are always in the same order: RWO,ROX,RWX.
+func GetAccessModesAsString(modes []PersistentVolumeAccessMode) string {
+	modes = removeDuplicateAccessModes(modes)
+	modesAsString := ""
+
+	if containsAccessMode(modes, ReadWriteOnce) {
+		appendAccessMode(&modesAsString, "RWO")
+	}
+	if containsAccessMode(modes, ReadOnlyMany) {
+		appendAccessMode(&modesAsString, "ROX")
+	}
+	if containsAccessMode(modes, ReadWriteMany) {
+		appendAccessMode(&modesAsString, "RWX")
+	}
+
+	return modesAsString
+}
+
+// GetAccessModesAsString returns an array of AccessModes from a string created by GetAccessModesAsString
+func GetAccessModesFromString(modes string) []PersistentVolumeAccessMode {
+	strmodes := strings.Split(modes, ",")
+	accessModes := []PersistentVolumeAccessMode{}
+	for _, s := range strmodes {
+		if s == "RWO" {
```

Comment:
Added Trim, just in case.  Because stringifying happened in the helper, I wasn't worried about decoding that same string without Trim.  Can't hurt, though.